css-pseudo-4/Overview.bs

```diff
@@ -724,10 +725,24 @@ Default UA Styles</h3>
     /* Represent default spelling/grammar error styling in an adjustable way */
     :root::spelling-error { text-decoration-line: spelling-error; }
     :root::grammar-error  { text-decoration-line: grammar-error; }
-    /* Highlight targeted text */
-    :root::target-text    { color: MarkText; background: Mark; }
   </pre>
 
+  Some highlight pseudo-elements
+  should have <dfn>default highlight colors</dfn>,
+  consisting of a 'color' and a 'background-color'.
+  For ''::selection'' they are
+  ''HighlightText'' and ''Highlight'',
+  while for ''::target-text'' they are
+  ''MarkText'' and ''Mark''.
+
+  Note: the painting behavior of highlight colors
+  means that default highlight colors
+  can't be expressed in terms of 'color' and 'background-color'
+  in [=UA style sheet=] rules.
```

Comment:
Can you explain why this wording is better than the original wording?

From the author's perspective (though you might implement it however you want), it's a special cascading rule, not a “painting” behavior, so I think the original wording is more understandable.

Also, you want `'color' and 'background-color' declarations` here.

Reply:
i had to reword it because i removed the preceding sentence

> The UA must use its own highlight colors for [::selection](https://drafts.csswg.org/css-pseudo-4/#selectordef-selection) only when neither [color](https://drafts.csswg.org/css-color-4/#propdef-color) nor [background-color](https://drafts.csswg.org/css-backgrounds-3/#propdef-background-color) has been specified by the author.

which in turn was removed because it’s now explained more thoroughly (and correctly) in #highlight-painting, but i see where you’re coming from. i’ll revert to the old text and correct the thing about ::selection.

Reply:
done in 26b6c2b32e30d2ef84cc48365939ce682eb17482